activemodel/CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
+*   Fix validation callbacks on multiple context
```

Comment:
How about wording here?

> *   Fix to working before/after validation callbacks on multiple contexts.

Reply:
Thanks for the comment. I've modified it :)